frontend/javascripts/test/geometries/skeleton.spec.ts

```diff
@@ -25,7 +25,7 @@ import {
 import { initializeAnnotationAction } from "oxalis/model/actions/annotation_actions";
 
 const skeletonCreator = () =>
-  new Skeleton((state: OxalisState) => getSkeletonTracing(state.annotation), true);
+  new Skeleton((state: OxalisState) => getSkeletonTracing(state.annotation)!, true);
```

Comment:
I used the Typescript `!` operator the enforce that a skeleton tracing is actually defined instead of checking for `null`. I think this should be fine in the test.

Reply:
I only see your comment now. in this particular case, there's no harm. however, code patterns tend to spread across the entire repo over time (copy&pasta etc), which is why I'd remove it altogether (e.g., newcomers might not know about the downsides of this).

Reply:
Done